interpreter/tokenizer.py

```diff
@@ -72,8 +72,11 @@ def get_token_type(cls, token):
             return TokenType.BINARY_OPERATOR
         if token.isdigit():
             return TokenType.NUM_CONST
+
+        # not sure if this is correct
         if token.startswith("\"") or token.startswith("'") or token.endswith("\"") or token.endswith("'"):
```

Comment:
I would write it like:
if (token.startswith("\"") and token.endswith("\"")) or (token.startswith("'") and token.endswith("'")):
this way if its between "" or '' it is a CharConst